Rlabkey/R/labkey.experiment.R

```diff
@@ -171,3 +171,30 @@ labkey.experiment.saveBatch <- function(baseUrl=NULL, folderPath, assayConfig =
 
 	return (fromJSON(response, simplifyVector=FALSE, simplifyDataFrame=FALSE))
 }
+
+labkey.experiment.saveRuns <- function(baseUrl=NULL, folderPath, protocolName, runList)
+{
+    baseUrl=labkey.getBaseUrl(baseUrl)
+
+    ## Validate required parameters
+    if (missing(folderPath)) stop (paste("A value must be specified for folderPath."))
+    if (missing(protocolName)) stop (paste("A value must be specified for protocolName."))
+    if (missing(runList)) stop (paste("A value must be specified for runList."))
+
+    ## normalize the folder path
+    folderPath <- encodeFolderPath(folderPath)
+
+    ## Now post form with runs object filled out
+    url <- paste(baseUrl, "assay", folderPath, "saveAssayRun.api", sep="")
```

Comment:
I get the following error when trying to test out this new Rlabkey function locally. Is there a typo here? Should this be "saveAssayRuns.api"?

<img width="1078" alt="Screen Shot 2019-09-16 at 10 11 05 AM" src="https://user-images.githubusercontent.com/6411206/64969868-57a3db00-d86a-11e9-889c-35fc1b014362.png">

Reply:
oh dang! I forgot to change it here.. I changed the action name from SaveRun to SaveRuns last Friday. 
Thanks for catching it.